spec/models/comment_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before :each do
+    @user = User.new(name: 'John Doe', photo: 'https://example.com/photo.jpg', bio: 'hola', posts_counter: 0)
+    @user.save
+    @post = Post.new(author: @user, title: 'Post title', text: 'body', comments_counter: 0, likes_counter: 0)
+    @post.save
+    @comment = Comment.new(author: @user, text: 'comment', post: @post)
+    @comment.save
+  end
+
+  it 'text always valid' do
+    @comment.text = 'comment'
+    expect(@comment).to be_valid
+  end
```

Comment:
- [ ] You currently have 17 out of 24 tests failing. Kindly take a look below and try to fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/178109723-cd4533a6-bb8d-4c48-a1e6-57489328a9ed.png)